docs/cookbook/templates.rst

```diff
@@ -80,6 +80,7 @@ template you are overriding:
    - ``%name%``  the specification name
    - ``%namespace%`` the specification namespace
    - ``%subject%`` the name of the class being specified
+   - ``%subject_class%`` the name of the subject class being specified
```

Comment:
The description is not really clear. Subject's description could be also clarified. How about:

```
- ``%subject%``the fully qualified name of the class being specified
- ``%subject_class%`` the name of the class being specified
```